components/ukm/METADATA

```diff
@@ -0,0 +1,20 @@
+name: "ukm"
+description:
+  "Filtered subtree at components/ukm."
+
+third_party {
```

Comment:
Could you leave the comment for approximate upstream branch / release ? e.g. m70 or such. Hash is great for exact pin, but it doesn't tell much to the human reading it